cmd/server/loader_basic.go

```diff
@@ -2,35 +2,42 @@ package main
 
 import (
 	"fmt"
-	pcgr "github.com/dgryski/go-pcgr"
-	mct "github.com/dormando/mctester"
 	"math/rand"
 	"time"
+
+	"github.com/dgryski/go-pcgr"
+	mct "github.com/memcached/mctester"
```

Comment:
These changes are related to Fei's PR as well. I did not make further changes to the server application.